CONTRIBUTING.md

```diff
@@ -13,19 +13,19 @@ Contributing to the HTML is slightly more complex. The main HTML generator is in
 Here's a step-by-step guide to get you set up for HTML contribution:
 
 1. **Install requirements:**
-   * Install a recent version of [HTML tidy](http://www.html-tidy.org/) (used for HTML validity testing). Here's how:
-     - On MacOS (should be version 5.6.0 by the date of this writing): `brew install tidy-html5`   
+   * Install a recent version of [HTML Tidy](http://www.html-tidy.org/) (used for HTML validity testing). Here's how:
+     - On MacOS (should be version 5.8 by the date of this writing): `brew install tidy-html5`   
      - On Debian / Ubuntu: `sudo apt-get install tidy`
-   * Install a recent version of [jq](https://github.com/stedolan/jq). Here's how:
+   * Install a recent version of [`jq`](https://github.com/stedolan/jq). Here's how:
      - On MacOS: `brew install jq`
      - On Debian / Ubuntu: `sudo apt-get install jq`
 
 2. **Set up for development:**
-   Clone the odoc repository, navigate into the new directory, and install dependencies with these commands:
+   Clone the `odoc` repository, navigate into the new directory, and install dependencies with these commands:
    ```
    git clone https://github.com/ocaml/odoc.git
    cd odoc
-   opam pin add --no-action odoc .
+   opam pin add --no-action `odoc` .
```

Comment:
You need to add `opam pin add --no-action odoc-parser .` before this line.

Reply:
I added this and ran that sequence again, but I'm still getting
```
[ERROR] Package conflict!
  * Missing dependency:
    - odoc-parser >= dev
    no matching version

No solution found, exiting
```

Reply:
I would just remove both `opam pin add` commands and change the `opam install` command to:

```
opam install --with-test --deps-only ./odoc-parser.opam ./odoc.opam
```

Reply:
Done! No longer getting that error. Thanks!